src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -475,7 +475,37 @@
                     </div>
                 </li>
             }
-        </ul>             
+        </ul>
+        <p class="h6">
+            <a data-bs-toggle="collapse" href="#apiRevisionOptionsCollapse" aria-expanded="true" aria-controls="approvalCollapse">API Revision Options&nbsp;&nbsp;<i class="fa-solid fa-ellipsis"></i></a>
+        </p>
+        @{
+            var apiRevisionOptionsCollapseState = " show";
+            if (Request.Cookies.ContainsKey("APIRevisionOptionsCollapse"))

```

Comment:
Cookie value is case sensitive. Ensure it is the same everywhere you rare using it.

Reply:
All changes made.